actions/testmo-run-submit-thread/action.yml

```diff
@@ -39,15 +43,19 @@ runs:
         fi
         ## submit results?
         SUCCESS=0
-        if [ ${REPORT} -eq 1 ]; then
+        if [[ ${REPORT} -eq 1 ]]; then
           echo "submitting results to TESTMO run ..."
           ## not checking testmo_url and token as this should be
           ## called between "create" and "complete"
-          npx testmo automation:run:submit-thread \
-            --instance ${TESTMO_URL} \
-            --run-id ${TESTMO_RUN_ID} \
-            --results ${RESULTS} \
-            -- ./.github/scripts/step-status ${{ inputs.step_status }}
+          submit_thread_options=( \
+            --instance ${inputs.testmo_url} \
```

Comment:
Why does the testmo_url needs to be input now and if it has to be it seems not added as input yet?

Reply:
line 4 defines this "testmo_url" input.
It is required because this command actually submits the thread to the TestMo server.

Reply:
I think the question is more so why are we changing this instead of using the pre-existing `--instance ${TESTMO_URL}` / `--run-id ${TESTMO_RUN_ID}` / `--results ${RESULTS}` since the action already takes these inputs and transforms them into environment variables.

Reply:
six of one, half dozen of the other?  It didn't seem appropriate to transform the new input into an env var, and I wanted to be consistent with usage w/in the bash code.  no big deal, I can change them back

Reply:
I always like as few changes as makes sense. If you think it makes sense to keep the direct references to the `inputs` then by all means keep that, but I would then expect that the `env` block in the step is cleaned up if we're not going to use the env vars it declares (after making sure they aren't used anywhere else, of course).

Reply:
understood.  i've reverted back to using the env vars.